apps/dashboard/app/javascript/packs/files/data_table.js

```diff
@@ -0,0 +1,441 @@
+import 'datatables.net';
+import 'datatables.net-bs4/js/dataTables.bootstrap4';
+import 'datatables.net-select';
+import 'datatables.net-select-bs4';
+import Handlebars from 'handlebars';
+import {EVENTNAME as CLIPBOARD_EVENTNAME} from './clip_board.js';
+import {EVENTNAME as FILEOPS_EVENTNAME} from './file_ops.js';
+import {EVENTNAME as SWAL_EVENTNAME} from './sweet_alert.js';
+
+export { CONTENTID, EVENTNAME };
+
+const EVENTNAME = {
+    getJsonResponse: 'getJsonResponse',
+    reloadTable: 'reloadTable',
+    goto: 'goto',
+};
+
+const CONTENTID = {
+    table: '#directory-contents',
+};
```

Comment:
Why not just make this a string so that the consumers don't have to call `.table`. 

```suggestion
const TABLE_ID = '#directory-contents';
```

Reply:
The initial reason was in case we had multiple ids in the future, but those ids would be tied to the different classes, so you have a good point.